packages/fxa-settings/src/components/HeaderLockup/index.tsx

```diff
@@ -10,71 +10,62 @@ import Avatar from '../Avatar';
 import { ReactComponent as Help } from './help.svg';
 import { ReactComponent as Bento } from './bento.svg';
 import { ReactComponent as Menu } from './menu.svg';
+import { useAccount } from '../../models';
 
-type HeaderLockupProps = {
-  avatarUrl: string | null;
-  primaryEmail: string;
-};
-
-export const HeaderLockup = ({
-  avatarUrl,
-  primaryEmail,
-}: HeaderLockupProps) => {
-  const right = (
-    <HeaderContentRight
-      {...{
-        avatarUrl,
-        primaryEmail,
-      }}
-    />
+export const HeaderLockup = () => {
+  const { avatarUrl, primaryEmail } = useAccount();
```

Comment:
I wonder if this should be called something like `getAccountCache` instead? `use` makes me think it's a hook.

Reply:
it is a hook

Reply:
I guess it returns the result of another hook 😛